onnxruntime/test/providers/provider_test_utils.cc

```diff
@@ -626,14 +626,15 @@ void OpTester::Run(
     const RunOptions* run_options,
     std::vector<std::unique_ptr<IExecutionProvider>>* execution_providers,
     ExecutionMode execution_mode,
-    const CustomOutputVerifierFn& custom_output_verifier) {
+    const CustomOutputVerifierFn& custom_output_verifier,
+    const Graph::ResolveOptions* resolve_options) {
```

Comment:
this could be a `const ResolveOptions&`, with a default parameter value of `{}`

Reply:
will try ...